doc/related_projects.rst

```diff
@@ -302,3 +302,21 @@ Domain specific packages
 
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
+
+Translations of scikit-learn documentation
+------------------------------------------
+
+The only official documentation for scikit-learn is the current English
+version. However, some translations are available aiming to improve scikit-learn
+dissemination among non-native English speakers. Those efforts are community
```

Comment:
Most of us are non-native English speakers, and yet we're here ;)

```suggestion
dissemination among non-English speakers. Those efforts are community
```

Reply:
> Most of us are non-native English speakers, and yet we're here ;)

So what? I'm sure 90% of our (many) discussions will be more easily resolved if you were Italian or I was French. I'm a (very bad) English speaker but I'm not native and this could make quite a difference. So if you don't mind I will let this one as is. :)

Reply:
My point is that non-native English speakers are *by definition* English speakers. It's just not their mother tongue.

The translations are primarily for people that don't speak English. These are not non-native English speakers, they're non-English speaker (or whatever a better term might be)

Reply:
> The translations are primarily for people that don't speak English. These are not non-native English speakers, they're non-English speaker (or whatever a better term might be)

Translations are extremely helpful for persons that already know the subject in their own language even if they speak English. Mapping semantic to concepts from one language to another is not so obvious, especially between languages having so different syntactical structures.

Reply:
I don't disagree with that.
I disagree with the claim that translations are for non-native English speakers. They're for people that either don't speak English, or for people that don't feel comfortable enough in English. These are not "non-native English speakers".

I'll stop arguing now.

Reply:
I think you both wan't to say the same thing, i.e Traductions are for everyone not comfortable enough with the english version of the docs. To express that, I think Nicolas' formulation is better suited. non-English speaking person means both people who don't speak english at all and people who don't speak english natively.

Reply:
Thanks Jeremie, still, I disagree about the wording, as I don't have a quantitative way to measure how well people could feel comfortable in English or not. I don't feel as my formulation could exclude anybody: it is objective. I would be happier myself with an Italian translation of this documentation because I discover everyday that I have already approached some concepts in Italian but they were 'just' formulated differently. And this is quite true for French-Italian mapping in my everyday conversations... yet, people find my French quite good. Anyway, It's time to merge this PR because it has been open for too long. Feel free to change one word and merge. Thanks.

Reply:
> I don't feel as my formulation could exclude anybody

"Non-native English speakers" excludes people that don't speak English at all. Again, by definition: **a non-native English speaker is someone who speaks English** - no matter how good or bad.

> Feel free to change one word and merge. Thanks.

Let's try to avoid these kind of comments please. I personally find it quite disrespectful to see reviews being met with "You can just do it yourself" kind of comments. It's not how we're supposed to work and collaborate. I've seen these a bunch of times, and quite frankly, it makes me wonder why I even bother sometimes.

Reply:
> by definition: a non-native English speaker is someone who speaks English - no matter how good or bad.

See how bad is my English? Understood. And modified as non-native English (probably bad, feel free to fix).

Reply:
I have added to the text the disclaimer of the linux documentation, which in my understanding explains what I meant to say.
@NicolasHug I'm sorry if I sound disrespectful in any way. My explanation was bad and you clearly did not understand what I was intending to say. Hope this is clear now. I'm resolving this conversation as the text has significantly changed.

Reply:
> you clearly did not understand what I was intending to say

For the record I understood what you meant to say, and like I said, I didn't disagree with it.